cmd/helpers.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+    "github.com/anaskhan96/soup"
+	"os/exec"
+    "runtime"
+)
+
+// fetchHTML - fetches and parses HTML from the given URL
```

Comment:
nit: function name is enough. hiphens add confusion. end with `full-stop`.

```suggestion
// fetchHTML fetches and parses HTML from the given URL.
```